maintenance/lsrd_stats.py

```diff
@@ -181,39 +176,10 @@ def db_prodinfo(dbinfo, begin_date, end_date):
               AND o.order_date::date >= %s
               AND o.order_date::date <= %s;''')
 
-    #counts = {'sr': 0,
-    #          'therm': 0,
-    #          'toa': 0,
-    #          'source': 0,
-    #          'meta': 0,
-    #          'custom': 0,
-    #          'evi': 0,
-    #          'msavi': 0,
-    #          'nbr': 0,
-    #          'nbr2': 0,
-    #          'ndmi': 0,
-    #          'ndvi': 0,
-    #          'savi': 0,
-    #          'cfmask': 0}
-
-    import psycopg2
-    from collections import OrderedDict
-    with DBConnect(**dbinfo) as db:
-        try:
-            db.cursor.execute(sql, [begin_date, end_date])
-            desc = db.cursor.description
-            
-            results = [
-                OrderedDict(zip([col[0] for col in desc], row))
-                for row in db.cursor.fetchall() ]
-        except psycopg2.Error as e:
-            raise Exception(e)
-        
-        #db.select(sql, (begin_date, end_date))
-        #for i, key in enumerate(sorted(counts.keys())):
-        #    counts[key] = int(db[0][i])
-
-    #return counts
+    with DBConnect(cursor_factory=psycopg2.extras.DictCursor, **dbinfo) as db:
```

Comment:
You COULD get the psycopg2 stuff completely out of this module if you want by variabling it out in dbconnect.  Then from here you could do ```with DBConnect(cursor_type=DBConnect.dict_cursor, **dbinfo) as db:```  Would require code to account for it in DBConnect.